proto/select.proto

```diff
@@ -36,11 +36,11 @@ message SelectResponse {
 	repeated Error warnings = 4;
 	repeated int64 output_counts = 5;
 	optional int64 warning_count = 6;
-	// It indicates the encode type of response.
-	optional EncodeType encode_type = 7 [(gogoproto.nullable) = false];
```

Comment:
Better to reserve tag 7.

https://developers.google.com/protocol-buffers/docs/proto3#reserved